docs/release-notes/7.17.asciidoc

```diff
@@ -2,14 +2,24 @@
 == 7.17
 
 [discrete]
-[[release-notes-7.17.1]]
-=== 7.17.1
+[[release-notes-7.17.2]]
+=== 7.17.2
 
 [discrete]
-[[bug-fixes-7.17.1]]
+[[bug-fixes-7.17.2]]
 ==== Bug fixes and enhancements
+* Fixes an {endpoint-sec} integration bug that prevented benign Windows files from being deleted under certain circumstances.
+* Backports the bug that prevented Endpoint Security 8.1.1 from being run on Windows 8.1 and Server 2012 R2. With this change, the primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc. (https://github.com/elastic/endpoint/issues/15[#15]).
```

Comment:
"backports the bug" sounds wrong 😄 

```suggestion
* Backports the fix for the bug that prevented Endpoint Security 8.1.1 from being run on Windows 8.1 and Server 2012 R2. With this change, the primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc. (https://github.com/elastic/endpoint/issues/15[#15]).
```

Reply:
Changing to the following as discussed over Slack:
To ensure Endpoint Security continues to run on all supported Windows versions, the primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc.